libs/website/builder/src/lib/navbar.interface.ts

```diff
@@ -12,5 +12,6 @@ export type BuilderNavbarProps = PropsWithChildren<
     loginUrl?: string
     profileUrl?: string
     subscriptionsUrl?: string
+    showSubscriptionsUrl?: boolean
```

Comment:
I guess okay for now, we need to find a better way to also add stuff etc to it in the future